after_success

```diff
@@ -113,7 +113,9 @@ elif [[ ${CI_OS_NAME} = linux ]]; then
       exit 0
     fi
 
-    generate_coverage_data
+    if [[ ";${DO_COVERAGE_ON_BRANCH};" == *";${TRAVIS_BRANCH};"* ]]; then
```

Comment:
CI_BRANCH is equal to TRAVIS_BRANCH. I can modify it if you want. 

The `*`is for example when you have `export DO_COVERAGE_ON_BRANCH="master;release"`